vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -13,6 +13,29 @@ import "IReactPropertyBag.idl";
 
 namespace Microsoft.ReactNative
 {
+
+  DOC_STRING(
+    "Used to represent the state of the React Native JavaScript instance")
+  enum LoadingState
+  {
+    DOC_STRING(
+      "The instance is loading the JavaScript bundle and initial instance setup. Calls to run JavaScript functions will be queued to run once the instance is fully loaded."
+    )
+    Loading = 0,
+    DOC_STRING(
```

Comment:
>DOC_STRING [](http://example.com/codeflow?start=4&length=10)

nit: consider to add an empty line before each DOC_STRING macro in this enum for readability.